tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -10,6 +10,147 @@ exports[`access_hidden_namespace.w 1`] = `
 
  
  
+Tests 1 failed (1)
+Test Files 1 failed (1)
+Duration <DURATION>"
+`;
+
+exports[`access_modifiers.w 1`] = `
+"error: Method \\"private_method\\" is not public but it's part of the \\"SomeInterface\\" interface
+   --> ../../../examples/tests/invalid/access_modifiers.w:40:3
+   |
+40 |   private_method() {}
+   |   ^^^^^^^^^^^^^^ Method \\"private_method\\" is not public but it's part of the \\"SomeInterface\\" interface
```

Comment:
optional - I'm not sure if this message makes it obvious why being part of the interface means it has to be public. Maybe we could reword this to `Interface members must be declared public` or `Method "foo" is private in "Foo" but not in "SomeInterface"`, or something like that.

```suggestion
40 |   private_method() {}
   |   ^^^^^^^^^^^^^^ Interface members must be declared public
```